pages/linux/at.md

```diff
@@ -0,0 +1,15 @@
+# at
+
+> Executes commands at a specified time.
+
+- Open `at` prompt to execute commands at specific time. When you're done, press Ctrl+D to save the schedule:
```

Comment:
I really like the combination -  "Open an `at` prompt to create a new set of scheduled commands, press Ctrl+D to save and exit:"
thank you guys for help, will push final version with all fixes.